### DISCUSSION TITLE
Spelling and grammatical mistakes

### DISCUSSION THREAD

**Discussion by coltonrobbins73** [biostars/biostar-handbook-issues#321] [General]
Hello Dr. Albert,

First off, the biostars handbook is one of the most impactful learning resources I have encountered, so thank you for your incredible effort over the years!

However, you might be aware that there are many spelling and grammatical mistakes throughout the handbook which significantly impact the clarity of the information. Perhaps this is already being resolved. However, if there are no immediate plans for resolution, I'm wondering if there might be an efficient method for me to document errors as I read through. Let me know, and I would be happy to help in any way that I can.

**Comment by ialbert** [comment]
Sorry about that, I am fully aware of it. 

I have come to believe that I have some sort of dyslexia that does not let me see an incorrect sentence at the time when I write it. I could reread it countless times - but as long as it is fresh in my brain, I read the projection in my brain instead of the actual words on paper. Very frustrating for me as well.

I can only see mistakes later, many hours or days later. And then I do notice myself that some sentences are so fundamentally broken it is cringy. 

For new content, I am using proofreading via Grammarly, and that helps immensely. I think the new chapters do not have the aggravating type of mistakes anymore.

The older content is a bit outdated, and I was planning on reworking it altogether and passing it through Grammarly at that time, though various things come up and push that back. 

If you really wanted to contribute, I could give you access to a fork of the book and you could edit the markdowns directly.

**Reply by coltonrobbins73** [reply]
I completely understand, it's a lot of content to work through as an independent publisher. The mistakes are minor and can always be resolved from the surrounding context but it just slows down the reading slightly so I thought it might help.

Yes, absolutely, I would be happy to run through the markdown files for you.

**Comment by ialbert** [comment]
Let me suggest a simpler and more streamlined alternative: collect the suggestions in a document, 

It could be a shared Google Doc (share it to istvan.albert at gmail.com) where you could point out the chapter and perhaps the start of the sentence. No other corrections are needed, and I'll rework those passages.

This would be immensely useful as one of the main bottlenecks in fixing the errors is that there is too much content that I need to get through. If I knew what to fix, it would massively quicker.

It is also important that it is not too onerous of a task for you either.
